packages/sitecore-jss/src/personalize/graphql-personalize-service.ts

```diff
@@ -40,7 +45,7 @@ type PersonalizeQueryResult = {
 
 export class GraphQLPersonalizeService {
   private graphQLClient: GraphQLClient;
-
+  private timeout?: number;
```

Comment:
Not required to define separate private variable, you can just use value provided from the `this,config`